File: api-gateway/src/test/java/org/chilly/api_gateway/containers/ContainersEnvironment.java
package org.chilly.api_gateway.containers;

import org.chilly.common.test.containers.EurekaTestContainer;
import org.testcontainers.junit.jupiter.Container;
import org.testcontainers.junit.jupiter.Testcontainers;

@Testcontainers
public class ContainersEnvironment {
    @Container
    public static EurekaTestContainer eurekaContainer = EurekaTestContainer.getInstance();
}

File: files-svc/src/test/kotlin/com/chilly/files/svc/containers/ContainersEnvironment.kt
package com.chilly.files.svc.containers

import org.chilly.common.test.containers.EurekaTestContainer
import org.chilly.common.test.containers.MinioTestContainer
import org.testcontainers.junit.jupiter.Container
import org.testcontainers.junit.jupiter.Testcontainers

@Testcontainers
open class ContainersEnvironment {

    companion object {
        @Container
        val minIO: MinioTestContainer = MinioTestContainer.getInstance()
        @Container
        val eurekaTestContainer: EurekaTestContainer = EurekaTestContainer.getInstance()
    }
}
